app/services/atmosphere/optimizer.rb

```diff
@@ -8,8 +8,6 @@ class Optimizer
     # :appliance_id DB id of the appliance that was created. Optimization only akes care of finding a vm or creating a new one for given appliance.
     def run(hint)
       satisfy_appliance(hint[:created_appliance]) if hint[:created_appliance]
-      terminate_unused_vms if hint[:destroyed_appliance]
-      scale(hint[:scaling]) if hint[:scaling]
```

Comment:
These 2 hints are not used anymore, instead services are used - so just a cleaning.